src/App.tsx

```diff
@@ -1,156 +1,213 @@
+/* eslint-disable max-len */
```

Comment:
Remove the 'eslint-disable max-len' comment, it's generally better to follow the project's coding style and line length restrictions. Break long lines into multiple lines if needed.